### ISSUE TITLE
JSON data seems to be parsed in chunks causing parsing to fail

### ISSUE THREAD

**Issue by haykam821** [AvaloniaUI/AvaloniaVSCode#75]
With version 0.0.25 of this extension, for a particularly large project, the solution JSON will be read incorrectly:

```
SyntaxError: Unterminated string in JSON at position 8192
    at JSON.parse (<anonymous>)
    at updateSolutionModel (avaloniateam.vscode-avalonia-0.0.25/out/services/solutionParser.js:62:23)
    at Socket.<anonymous> (avaloniateam.vscode-avalonia-0.0.25/out/services/solutionParser.js:99:13)
    at Socket.emit (node:events:513:28)
```

The entire JSON file at the path printed by 'Solution data path path' is valid.

This issue seems to be caused by the stream emitting chunks of data of size 8192, where each chunk won't be valid JSON on its own. I believe a solution would be to append JSON to a string until the entire stream is read (`end` event) and then parsed.

https://github.com/AvaloniaUI/AvaloniaVSCode/blob/fc4fc2896b5a94804bd51b47be86d37827b16489/src/vscode-avalonia/src/services/solutionParser.ts#L112-L116

**Comment by prashantvc** [comment]
How big is your solution? Could you share more details please?

**Comment by BinToss** [comment]
@haykam821 Please check if  #111 solved the issue.

**Comment by MrJul** [comment]
This issue was normally solved by #111.
Please ping me to reopen if that wasn't the case.
